content/docs/uischema/rules.mdx

```diff
@@ -100,6 +103,20 @@ The following rule evaluates to true if the `counter` property is `1 <= counter
 }
 ```
 
+This rule is true if the `counter` property exists *and* is larger than 1.
+This is in contrast with the previous rule, which will evaluate as true if the `counter` property is undefined.
```

Comment:
```suggestion
`SHOW` applies if the `counter` property exists *and* is larger than 1.
If `failWhenUndefined` is not given or set to `false`, `SHOW` will also apply in case the `counter` property does not exist.
```

Reply:
This one is a bit confusing. One of the example rules claims to apply the SHOW effect if `counter` is 1 <= counter <= 10`, and I wrote a new example to contrast which specifically contains `failWhenUndefined` as part of the Rule object. I am inclined to believe that an explicit example is helpful to have, rather than repeating the "Rule Condition" section.

Incidentally, all of the other examples use the "evaluates to true" wording, so if you want to instead use "SHOW applies," all of them should be fixed.

Reply:
Good point, let's use the same terminology for now :+1: 

```suggestion
This rule evaluates to true if the `counter` property exists *and* is larger than 1.
This is in contrast with the previous rule, which will evaluate to true if the `counter` property is undefined.
```